app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,95 @@
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&family=Montserrat:wght@400;700&display=swap");
+
+body {
+  font-family: "Montserrat", sans-serif;
+  background-color: #f5f6f8;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+.main-text {
+  font-family: "Dancing Script", cursive;
+  font-size: 4rem;
+  font-weight: 400;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 10px;
+}
+
+.login-btns {
+  padding: 20px;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.login-btns * {
+  width: 100%;
+  height: 40px;
+}
+
```

Comment:
- [ ] Well done styling your project however you should follow the design and using the same colors for the buttons and the header.